src/main/scala/com/gd/twitteranalytics/StreamingTweetsJob.scala

```diff
@@ -0,0 +1,50 @@
+package com.gd.twitteranalytics
+
+import com.gd.twitteranalytics.TweetsTransformer._
+import com.gd.twitteranalytics.TweetsDataLoader._
+import com.gd.twitteranalytics.util.AppConfigReader
+import org.apache.log4j.Logger
+import org.apache.spark.SparkConf
+import org.apache.spark.streaming.dstream.DStream
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import java.util.{Date => JavaDate}
+
+object StreamingTweetsJob {
+
+  val log = Logger.getLogger(StreamingTweetsJob.getClass.getName)
+
+  def main(args: Array[String]): Unit = {
+      val authKeys = AppConfigReader.getTwitterAuthKeys
+      if (ConfigValidator.isConfValid(authKeys)) {
+        val Array(savePath, tweetsLangFilter, hashTagsFilter) = AppConfigReader.getAppConfigurables
+        val dateColumn = "event_date"
+        val ssc = setSparkStreamingContext
+
+        TweetsDataReader.configureTwitter(authKeys)
+
+        val englishTweets = TweetsDataReader.getTweets(ssc, Array(hashTagsFilter), tweetsLangFilter)
+        val tweetsInfo = getCreationDateAndStatus(englishTweets)
+        processTweetsInfo(tweetsInfo, savePath, dateColumn)
+
+        ssc.start
+        ssc.awaitTermination()
+      }
+      else
+        System.exit(1)
+  }
+
+  def setSparkStreamingContext():StreamingContext = {
```

Comment:
please add a TODO for setting up checkpointing: 
https://spark.apache.org/docs/latest/streaming-programming-guide.html#checkpointing